.github/workflows/CI.yml

```diff
@@ -38,19 +38,10 @@ jobs:
         with:
           submodules: true
 
-      # Git functionality in CMAKE file does not work with given ubuntu image. Therefore, handling it here.
```

Comment:
This is no longer issue so I removed. All patches should be specified directly in build system.